pages/hooks/useFetchComments.ts

```diff
@@ -9,7 +9,7 @@ const getComments = async () => {
 
 export const useFetchComments = () => {
   return useQuery({
-    queryKey: [queryKeys.COMMENT], 
+    queryKey: [queryKeys.GET_COMMENT],
```

Comment:
Your function says `FetchComments` but your query key says `GET_COMMENT`.

I think that is a little confusing and/or misleading